src/pbkdf2_hmac_common_plug.c

```diff
@@ -638,6 +638,22 @@ char *pbkdf2_hmac_sha256_prepare(char *fields[10], struct fmt_main *self) {
 	static char Buf[PBKDF2_SHA256_MAX_CIPHERTEXT_LENGTH + 1];
 	char tmp[43+1], *cp;
 
+	// MIME Base64, optionally with escaped slashes. Trailing '=' is also optional.
+	if (!strncmp(fields[1], PBKDF2_SHA256_FORMAT_TAG, PBKDF2_SHA256_TAG_LEN)) {
+		if (strchr(fields[1], '\\') || strchr(fields[1], '=') || strchr(fields[1], '+')) {
+			strcpy(Buf, fields[1]);
+			while ((cp = strchr(Buf, '\\')))
+				memmove(cp, cp + 1, strlen(cp));
+			while ((cp = strchr(Buf, '=')))
+				memmove(cp, cp + 1, strlen(cp));
```

Comment:
I'd speed these up a bit by doing a `cp = Buf` before the loops and then `while ((cp = strchr(cp, '\\')))`, so that only the remaining portion of string is searched.